go.mod

```diff
@@ -177,3 +177,5 @@ require (
 )
 
 replace github.com/opencontainers/runc => github.com/opencontainers/runc v1.1.1-0.20220617142545-8b9452f75cbc
+
+replace github.com/containers/storage => github.com/giuseppe/storage v1.19.2-0.20230130105941-634c638e5c48
```

Comment:
Isn't this already merged?

Reply:
it is failing on freebsd, so I'll need another patch